readme.txt

```diff
@@ -0,0 +1 @@
+hello world
```

Comment:
Please start with capital charcter? like "Hello...
And please add dot at the end of sentence.

Reply:
complete